consensus/consensus.go

```diff
@@ -30,6 +31,15 @@ import (
 	"github.com/autonity/autonity/rpc"
 )
 
+// Place consensus message codes in protocol to break dependency circle in between tendermint and fault detectors.
+
+// setting for Autonity accountability protocol, they are a part of consensus.
```

Comment:
Saying they are part of consensus is a bit misleading imo
```suggestion
// setting for Autonity accountability protocol
```

Reply:
I didn't write this comment, I'm going to remove it.